source/_patterns/03-layouts/regions/breadcrumb/breadcrumb.twig

```diff
@@ -1,5 +1,7 @@
 <div class="l-breadcrumb">
-  <div class="l-constrain">
+  {# l-constrain commented out to accomodate full-bleed default breadcrumb
```

Comment:
This seems overkill. Should we just create a parent component where we can enable and disable the constrain wrapper? Then all the regions can use it.